### ISSUE TITLE
Selected images in multiselect mode lost somtimes!

### ISSUE THREAD

**Issue by majidkabir** [ParkSangGwon/TedBottomPicker#32]
When I use the camera icon to capture picture and add to selected images in multi-select mode, sometimes screen rotate after camera saved the picture. In this state, your library onCreate called again and selectedUriList lost! 
I added this code for saving this variable in onSaveInstanceState and in setupDialog check if it's null then create a new instance.
It's worked just for saving the array variable, but the view for this selected images must recreate!

**Comment by majidkabir** [comment]
In setupDialog 
```
        if (selectedUriList == null) {
            selectedUriList = new ArrayList<>();
        }
```

**Comment by majidkabir** [comment]
I change setupDialog from
```
selectedUriList = new ArrayList<>();
```
to
```
        if (selectedUriList == null) {
            selectedUriList = new ArrayList<>();
            Log.d(TAG, "setupDialog:" + selectedUriList.size());
        } else {
            for (Uri uri:selectedUriList){
                addUri(uri);
            }
        }
```
and it works like a charm.

**Comment by ParkSangGwon** [comment]
@majidkabir This issue solved at v1.0.10
